pkg/storage/stores/indexshipper/uploads/metrics.go

```diff
@@ -0,0 +1,25 @@
+package uploads
```

Comment:
old metrics.go for reference https://github.com/grafana/loki/blob/440fb7d74cad481bebb222b21fc063fa56c2f5ac/pkg/storage/stores/shipper/uploads/metrics.go
`openExistingFileFailuresTotal` is only relevant for `boltdb-shipper` so it is already added to it [here](https://github.com/grafana/loki/blob/6ac955cc7d395874ce60516d182678542415bca3/pkg/storage/stores/shipper/index/metrics.go#L14)